spec/widgets/auto-style/style-utils.spec.js

```diff
@@ -0,0 +1,35 @@
+var StyleUtils = require('../../../src/widgets/auto-style/style-utils.js');
+
+describe('src/widgets/auto-style/style-utils', function () {
+  describe('.changeStyle', function () {
+    it('should change style attribute', function () {
+      var cartocss = '#layer {marker-line-color: white; marker-fill: #adadad;}';
+      var attr = 'marker-fill';
+      var newStyle = ' marker-fill: blue;';
+
+      var expected = '#layer {marker-line-color: white; marker-fill: blue;}';
+
+      expect(StyleUtils.changeStyle(cartocss, attr, newStyle)).toBe(expected);
+    });
+
+    it('should change style and remove duplicated', function () {
+      var cartocss = '#layer {marker-line-color: white; marker-fill: #adadad; marker-fill: #fafafa;}';
+      var attr = 'marker-fill';
+      var newStyle = ' marker-fill: blue;';
+
+      var expected = '#layer {marker-line-color: white; marker-fill: blue;}';
+
+      expect(StyleUtils.changeStyle(cartocss, attr, newStyle)).toBe(expected);
+    });
+
+    it('should change style, remove duplicated and remove empty braces', function () {
+      var cartocss = '#layer {marker-line-color: white; marker-fill: #adadad; [me>gasol] { marker-fill: #fafafa; }}';
```

Comment:
Would:
`var cartocss = '#layer {marker-line-color: white;  [me>gasol] { marker-fill: #fafafa; }; marker-fill: #adadad; }'` work too?

What about:

`'#layer {marker-line-color: white; marker-fill: #adadad; [me>gasol] { marker-fill: #fafafa; marker-width: 200}}'` ? (we could add a test to ensure layers that are not empty are not removed)